### PR TITLE
feat(admin): change status of group-resource assigment

### PR DESCRIPTION
**Pull request by bodnara**
* group-resource assignment can be activated or deactivated from both group and resource lists

### CODE REVIEW

**Review by HejdaJakub — CHANGES_REQUESTED** [review]
The code seems to be ok, I have only one thing. If I want to change status from `INACTIVE` to `ACTIVE`, there is no default sync/async option selected and the submit button is not disabled, so I can submit the dialog without any selected option. I would prefer to choose one of them as the default selected option.
image

**Review by xkureck — CHANGES_REQUESTED** [review]
image
I want some description of what would happen if I change the status

image
Nothing is selected but I can still submit the dialog, if something would be preselected that would be great. 
image
The description of what is asynchronous/synchronous assigment is great but I still dont have any idea what that means.

**Review by HejdaJakub — COMMENTED** [review]
Loading spinner would look better in the middle of the dialog.
image

File: libs/perun/dialogs/src/lib/change-group-resource-assigment-dialog/change-group-resource-assigment-dialog.component.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
+<h1 mat-dialog-title>{{'SHARED_LIB.PERUN.COMPONENTS.CHANGE_GROUP_RESOURCE_ASSIGMENT_DIALOG.TITLE' | translate}}</h1>
+<mat-spinner *ngIf="loading"></mat-spinner>
+<div mat-dialog-content *ngIf="!loading">
+  <h5 class="mb-2">
```
Comment (xkureck) [inline libs/perun/dialogs/src/lib/change-group-resource-assigment-dialog/change-group-resource-assigment-dialog.component.html:4]:
```suggestion
  <p class="mb-4">
```

**Review by xkureck — CHANGES_REQUESTED** [review]

**Comment by xkureck** [comment]
> Loading spinner would look better in the middle of the dialog.
> image

agree with jakub, also some appropriate colour would be nice

File: libs/perun/dialogs/src/lib/change-group-resource-assigment-dialog/change-group-resource-assigment-dialog.component.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    }, () => this.loading = false);
+  }
+
+  onCancel() {
+    this.dialogRef.close(false);
+  }
+
+  onSubmit() {
+    this.loading = true;
+    if(this.status==="ACTIVE"){
```
Comment (xkureck) [inline libs/perun/dialogs/src/lib/change-group-resource-assigment-dialog/change-group-resource-assigment-dialog.component.ts:54]:
```suggestion
    if(this.status === "ACTIVE") {
```

File: libs/perun/dialogs/src/lib/change-group-resource-assigment-dialog/change-group-resource-assigment-dialog.component.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
+<div class="{{theme}}">
```
Comment (xkureck) [inline libs/perun/dialogs/src/lib/change-group-resource-assigment-dialog/change-group-resource-assigment-dialog.component.html:1]: please check the format of this html, add spaces where it should be, remove them where they shouldnt be

**Review by xkureck — APPROVED** [review]
just repair this little code readibility upgrades and then merge it

**Review by HejdaJakub — APPROVED** [review]

**Pull request merged**